mmaction/models/losses/cross_entropy_loss.py

```diff
@@ -20,13 +36,30 @@ def _forward(self, cls_score, label, **kwargs):
         Returns:
             torch.Tensor: The returned CrossEntropy loss.
         """
+        if self.class_weight is not None:
+            kwargs['weight'] = self.class_weight.to(cls_score.device)
```

Comment:
assert that `weight` do not present in kwargs before this line?

Reply:
It does not present elsewhere. But adding an assert line here is a good idea.